tools/build.py

```diff
@@ -443,17 +444,20 @@ def build_jerry(option):
     # Turn off LTO for jerry bin to save build time.
     cmake_opt.append('-DENABLE_LTO=OFF')
 
+    # Turn on snapshot
+    cmake_opt.append('-DFEATURE_SNAPSHOT_SAVE=ON')
+    cmake_opt.append('-DFEATURE_SNAPSHOT_EXEC=ON')
```

Comment:
Do we need snapshot execution? The jerry command line tool is only needed to _generate_ (i.e., save) snapshot, isn't it?